apps/server/src/services/schemaService.ts

```diff
@@ -30,37 +32,3 @@ export function validate(schema: Schema, references: References): { valid: boole
 
 	return parseResult.success ? { valid: true } : { valid: false, errors: parseResult.error };
 }
-
-/**
- * String formatting of values provided as scripts. This will normalize the formatting of newline characters,
- * All instances of `/r/n` will be converted to `/n`
- * @param script
- * @returns
- */
-function normalizeScript(input: string | string[]) {
-	const normalize = (script: string) => script.replace(/\r\n/g, '\n');
-
-	if (typeof input === 'string') {
-		return normalize(input);
-	} else {
-		return input.map(normalize);
-	}
-}
-
-export function normalizeSchema(schema: Schema): Schema {
```

Comment:
Only script fields were normalized. We have removed scripts from the Lectern Dictionary meta-schema in this PR.